src/window.rs

```diff
@@ -203,6 +204,13 @@ impl Window {
                         let notification = gio::Notification::new(&title);
                         notification.set_body(Some(&body));
 
+                        if let Some(avatar) = chat.avatar() {
+                            let local = &avatar.0.local;
+                            if let Ok(texture) = Texture::from_filename(&local.path) {
```

Comment:
You should probably check if the file is downloaded first.

Also, we should probably try to download it later and update the notification with the downloaded photo when it's completed. You can leave a TODO comment about that if you don't feel to do it in this PR.